cpp/src/utilities/wrapper_types.hpp

```diff
@@ -348,6 +330,120 @@ using date32 = detail::wrapper<gdf_date32, GDF_DATE32>;
 
 using date64 = detail::wrapper<gdf_date64, GDF_DATE64>;
 
+using bool8 = detail::wrapper<gdf_bool, GDF_BOOL>;
+
+// This is necessary for global, constant, non-fundamental types
+#define HOST_DEVICE_CONSTANT
+#ifdef __CUDA_ARCH__
+__device__ __constant__ static bool8 true_v{gdf_bool{1}};
+
+__device__ __constant__ static bool8 false_v{gdf_bool{0}};
+#else
+static constexpr bool8 true_v{gdf_bool{1}};
+static constexpr bool8 false_v{gdf_bool{0}};
+#endif
+
+// Wrapper operator overloads for cudf::bool8 
+namespace detail {
+
+// For some reason, get a multiple definition error if this is defined.
+// But it would be nice for the output of cudf::bool8 in ostreams to be 
+// more useful than an int8 value...
+/*std::ostream& operator<<(std::ostream& os, cudf::bool8 const& w) 
```

Comment:
Likely because it needs to be marked `inline`.

Reply:
Ah, thanks for catching that. To overload it I of course removed the template, which means it was no longer inlined.